tests/e2e/support/objects/app-admin-settings/groups/actions.ts

```diff
@@ -32,3 +37,48 @@ export const getDisplayedGroups = async (args: { page: Page }): Promise<string[]
   }
   return groups
 }
+
+export const selectGroup = async (args: { page: Page; uuid: string }): Promise<void> => {
+  const { page, uuid } = args
+  const checkbox = await page.locator(util.format(groupCheckboxSelector, uuid))
+  const checkBoxAlreadySelected = await checkbox.isChecked()
+
+  if (checkBoxAlreadySelected) {
+    return
+  }
+  await checkbox.click()
+}
+
+export const deleteGroupUsingContextMenu = async (args: {
+  page: Page
+  uuid: string
+}): Promise<void> => {
+  const { page, uuid } = args
+  await page.locator(util.format(groupIdSelector, uuid)).click()
+  await page.locator(deleteBtnContextMenu).click()
+
+  await Promise.all([
+    page.waitForResponse(
+      (resp) =>
+        resp.url().endsWith(encodeURIComponent(uuid)) &&
+        resp.status() === 204 &&
+        resp.request().method() === 'DELETE'
+    ),
+    await page.locator(actionConfirmButton).click()
+  ])
+}
+
+export const deleteGrouprUsingBatchAction = async (args: { page: Page }): Promise<void> => {
+  const { page } = args
+  await page.locator(deleteBtnBatchAction).click()
+
+  await Promise.all([
+    page.waitForResponse(
+      (resp) =>
+        resp.url().includes('groups') &&
+        resp.status() === 204 &&
+        resp.request().method() === 'DELETE'
+    ),
+    await page.locator(actionConfirmButton).click()
+  ])
+}
```

Comment:
Can we make these function into one `deleteGroup` and sitch the action instead?

Reply:
I wouldn't combine them because I need check all DELETE requests and 

see https://github.com/owncloud/web/issues/8648#issuecomment-1479409167

Reply:
I tried to do it but couldn't 
maybe you have ideas how?
see https://github.com/owncloud/web/compare/deleteGroup...checkSeveralResponses?expand=1

Reply:
anyway, I suggest to do it in another PRs

Reply:
okay we can do it in next PR if it's possible